acceptance-tests/src/test/java/org/eclipse/collections/impl/map/mutable/UnifiedMapAcceptanceTest.java

```diff
@@ -1580,6 +1580,7 @@ private void runGetTest(Map<CollidingInt, String> map, String mapName, int size)
             long runTimes = System.nanoTime() - startTime;
             LOGGER.info("{} get: {} ns per get on map size {}", mapName, (double) runTimes / (double) n / size, size);
         }
+        //noinspection AssignmentToMethodParameter
```

Comment:
This one has so many suppressions, I'd rather treat the inspection as a warning or turn it off.